tests/x11/reboot_gnome.pm

```diff
@@ -40,12 +40,19 @@ sub run() {
 
         send_key "ret";
 
-        wait_still_screen 8, 8;    # wait 8 seconds to make authentication window disappear after successful authentication
-        if (check_screen 'reboot-auth', 2) {
-            record_soft_failure 'bsc#981299';
-            send_key_until_needlematch 'generic-desktop', 'esc',             7, 10;    # close timed out authentication window
-            send_key_until_needlematch 'logoutdialog',    'ctrl-alt-delete', 7, 10;    # reboot
-            assert_and_click 'logoutdialog-reboot-highlighted';
+        # workaroung for bsc#981299
+        # run only on qemu backend, reset consoles on svirt to activate console/screen after reboot
+        if (check_var('BACKEND', 'qemu')) {
+            wait_still_screen 8, 8;    # wait 8 seconds to make authentication window disappear after successful authentication
```

Comment:
check_screen has to check if the reboot "failed", if the authentication window is found then it will try to reboot again
workaround is executed only on qemu backend, reset_consoles is done for svirt otherwise there was black screen